### PR TITLE
Add configuration properties for automatic envelope processing

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1126

### Change description ###

Add configuration properties required for automatic envelope processing - case creation, case update and case search.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -91,6 +94,9 @@ service-config:
         - formType: PERSONAL
           ocrFields:
             - last_name
+      auto-case-creation-enabled: ${AUTO_CASE_CREATION_ENABLED_BULKSCANAUTO:false}
+      auto-case-update-enabled: ${AUTO_CASE_UPDATE_ENABLED_BULKSCANAUTO:false}
+      case-definition-has-envelope-ids: ${CASE_DEFINITION_HAS_ENVELOPE_IDS_BULKSCANAUTO:false}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml:99]: normally first PR can be `AUTO_CASE_CREATION_ENABLED_BULKSCANAUTO` `AUTO_CASE_UPDATE_ENABLED_BULKSCANAUTO` and `CASE_DEFINITION_HAS_ENVELOPE_IDS_BULKSCANAUTO` can go in values yaml with false value and then override wherever needed. 
This way you get the flexibility of changing it per environment and also your chart config goes before your code usage.

Comment (lgonczar) [reply src/main/resources/application.yaml:99]: I'm not sure I understand what you mean.

Comment (lgonczar) [reply src/main/resources/application.yaml]: Done

Comment (nitinprabhu) [reply src/main/resources/application.yaml]: thanks @lgonczar

**Pull request merged**